editor/inserter/menu.js

```diff
@@ -245,12 +276,13 @@ class InserterMenu extends Component {
 
 	switchTab( tab ) {
 		this.setState( { tab: tab } );
+		this.setSearchFocus();
```

Comment:
Let's remove this for now to not mess with focus. We'll need another pass of keyboard flows here once things settle.